MediaBrowser.Controller/Library/TVUtils.cs

```diff
@@ -10,7 +10,7 @@ public static class TVUtils
         /// <summary>
         /// The TVDB API key
         /// </summary>
-        public static readonly string TvdbApiKey = "72930AE1CB7E2DB3";
+        public static readonly string TvdbApiKey = "OG4V3YJ3FAP7FP2K";
```

Comment:
is is yet another key replacement, or did we miss it last time?

Reply:
Isn't that reverting 9da5d3acc29319f78c63f32e29048fd478d8953a back from @drakus72 to the Jellyfin API key?

Reply:
This change is fine, as @liggy1 says, it’s going back to our own key, which is v2 only.